web/src/index.tsx

```diff
@@ -6,7 +6,7 @@
 import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 
-import "@fontsource/inter/variable.css";
+import "@fontsource/inter";
```

Comment:
`import "@fontsource-variable/inter";`
then in index.css change so the first font-family line starts like this (keep both variants):
`font-family: "InterVariable", "Inter Variable",`